GTG/gtk/backends/addpanel.py

```diff
@@ -170,7 +172,16 @@ def on_combo_changed(self, widget=None):
         """
         backend_name = self.combo_types.get_selected()
         if backend_name is None:
+            if 'backend_caldav' in inactive_modules:
+                markup = '<big>Error: Python package \'caldev\' not installed.</big>'
+                self.label_name.set_markup(markup)
+                return
+
+
+            markup = '<big>Error: An unknown backend could not be loaded.</big>'
```

Comment:
```suggestion
            markup = '<big>{}</big>'.format(_('Error: An unknown backend could not be loaded.'))
```

Make it translatable.